packages/server/postgres/queries/src/insertPollWithOptionsQuery.sql

```diff
@@ -0,0 +1,20 @@
+/*
+  @name insertPollWithOptionsQuery
+  @param poll -> (createdById, discussionId, teamId, meetingId, threadSortOrder, title)
+  @param pollOptions -> ((title)...)
+*/
+WITH poll AS (
+   INSERT INTO "Poll" (
+       "createdById",
+       "discussionId",
+       "teamId",
+       "meetingId",
+       "threadSortOrder",
+       "title"
+   ) VALUES :poll RETURNING *
+), pollOptionsData AS (
+  SELECT "title", (SELECT id from poll AS "pollId") FROM (VALUES :pollOptions) AS pollOptionsData(title) 
```

Comment:
this is the line that caused all the headaches i bet!

Reply:
Oh yeah. One thing I've learned about writing SQL is that you don't know the solution, start with a super simple query, data and then go towards the end goal. SQL errors are basically useless so it's hard to find out what's wrong with the query.